.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -0,0 +1,10 @@
+name: "Validate Gradle Wrapper"
+on: [pull_request]
```

Comment:
you have to change to

```
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```

to see result of PR in overview. But I'll stop to waste my time here